airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py

```diff
@@ -136,6 +142,66 @@ def test_two_sequential_reads(
                 record_value, state_value, threshold_days
             ), f"Second incremental sync should produce records older or equal to cursor value from the state. Stream: {stream_name}"
 
+    def test_read_sequential_slices(
+        self, inputs: IncrementalConfig, connector_config, configured_catalog_for_incremental, cursor_paths, docker_runner: ConnectorRunner
+    ):
+        """
+        Incremental test that makes an initial API request without a state checkpoint. Then we partition the results by stream and
+        slice checkpoints resulting in batches of messages that look like:
+        <state message>
+        <record message>
+        ...
+        <record message>
+
+        Using these batches, we then make API requests using the state message and verify the the correctness of the messages in the
+        response.
+        """
+        if inputs.skip_new_incremental_tests:
+            pytest.skip("Skipping new incremental test based on acceptance-test-config.yml")
```

Comment:
When I used `pytest.skip` within another SAT test I found it was also marking as skipped my unit test that tested the skipping. You may want to mock it and add a `return` here.

https://github.com/airbytehq/airbyte/blob/7322b86546e97d3342a4b1409e084dc84d22e6f3/airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py#L414-L416

Reply:
were you able to mock the `pytest.skip` method in your tests? i was playing around with some variations of mocking, but haven't been able to override the behavior to just skip and return
```
pytest_mock = MagicMock()
pytest_mock.skip.return_value = ""
```

Reply:
I was! Here's an example: https://github.com/airbytehq/airbyte/blob/957a0be226b65d0680a324124beb52179cd33adf/airbyte-integrations/bases/source-acceptance-test/unit_tests/test_core.py#L240

Reply:
sweet thanks! and yup already added the return below, but will add the mock in now as well